src/software/logger/logger.h

```diff
@@ -30,8 +38,30 @@ class LoggerSingleton
     LoggerSingleton()
     {
         logWorker = g3::LogWorker::createLogWorker();
+        // robot diagnostics logs are in
+        // bazel-out/k8-fastbuild/bin/software/gui/robot_diagnostics/robot_diagnostics_main.runfiles/__main__/software
+        // full system logs are in
+        // bazel-out/k8-fastbuild/bin/software/full_system.runfiles/__main__/software
```

Comment:
1. I'm getting 
```
FILE ERROR:  could not open log file:[/software/thunderbots.log]
		 std::ios_base state = 4
FILE ERROR:  could not open log file:[/software/thunderbots_filtered.log]
		 std::ios_base state = 4
```
when I run full_system_main and standalone_simulator_main. I think it's trying to access the root directly. Maybe we can try the /tmp directory?

2. It looks like `bazel-out/k8-fastbuild/bin/software/full_system.runfiles/__main__/software` is a binary file, because I can't read anything there.

3. Can we save logs from tests as well, for example, simulated_tests?

Reply:
1. I think this was because of a typo. Can you try it now?
2. oops, it's supposed to a directory, added a "/" to clarify
3. added a comment for that